server/mscalendar/settings.go

```diff
@@ -38,6 +38,13 @@ func NewSettingsPanel(bot bot.Bot, panelStore settingspanel.PanelStore, settingS
 		store.UpdateStatusSettingID,
 		settingStore,
 	))
+	settings = append(settings, settingspanel.NewBoolSetting(
+		store.ReceiveNotificationsWhileOnMeeting,
+		"Receive notifications while on meetings",
+		"Do you want to still receive Mattermost notifications while you are on a meeting?",
```

Comment:
I think they should be told explicitly what status they will be set to when they are asked this question. "If you select yes, we will set your status to `Away`, otherwise we will set it to `Do Not Disturb`.